package.json

```diff
@@ -15,7 +15,9 @@
     "react-router-dom": "^5.3.0",
     "react-scripts": "4.0.3",
     "react-transition-group": "^4.4.2",
+    "sass": "^1.42.1",
```

Comment:
I don't this `sass` is needed, looks like plain `css` to me? Probably best as it keeps it more inclusive for people

Reply:
Ok, I'll update it to plain css

Reply:
Sass removed